app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -75,12 +78,20 @@ export default function Overview({ user }: OverviewProps) {
         {user.city}
       </Typography>
       <Divider />
-      {user.userId === currentUserId && (
+      {user.userId === currentUserId ? (
         <CardActions className={classes.cardActions}>
           <Button component={Link} to={editProfileRoute}>
             {EDIT_PROFILE}
           </Button>
         </CardActions>
+      ) : (
+        <CardActions className={classes.cardActions}>
+          {mutationError && <Alert severity="error">{mutationError}</Alert>}
+          <AddFriendButton
+            userId={user.userId}
+            setMutationError={setMutationError}
+          />
+        </CardActions>
```

Comment:
There should be a check here for `user.friendshipStatus`, showing this on `NOT_FRIENDS`, maybe a grayed out pending button for `PENDING` (actually maybe nothing for now, that will be quicker) and nothing for `FRIENDS`